### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by jas11**

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
The files changed require a buildbot test. Buildbot has started.

**Comment by buildbot-princeton** [comment]
Your changes passed the buildbot test. See [build 284](http://specfem3d-globe-buildbot.princeton.edu/builders/specfem3d_globe_builder/builds/284).

File: src/specfem3D/read_adjoint_sources_ASDF.f90 [hunk 1]
```diff
@@ -130,7 +120,7 @@ subroutine check_adjoint_sources_ASDF(irec, nadj_sources_found)
 
     ! updates counter for found files
     nadj_sources_found = nadj_sources_found + 1
-
+
```
Comment (QuLogic) [inline src/specfem3D/read_adjoint_sources_ASDF.f90]: Try to avoid adding whitespace as then BuildBot needs to make an extra commit fixing it.

**Comment by komatits** [comment]
accept

On 10/03/2016 00:14, buildbot-princeton wrote:
> Your changes passed the buildbot test. See build 284
> <http://specfem3d-globe-buildbot.princeton.edu/builders/specfem3d_globe_builder/builds/284>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d_globe/pull/480#issuecomment-194559993>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr

**Pull request merged**